### DISCUSSION TITLE
Alternate Method of Invisibility?

### DISCUSSION THREAD

**Discussion by Totoqoe** [winstxnhdw/lc-hax#404] [Ideas]
When joining with antikick, there's a chance that your character model itself goes invisible and does not update its position unless you use a teleport of some kind (like a door or the teleporters).  
Compared to the normal invis command, this sort of invisibility is more useful, especially since you can use your voice chat and have others hear you, which can lead to some funny trolling.
![image-19](https://github.com/winstxnhdw/lc-hax/assets/158114082/cdeaa25f-6058-4785-896c-d060152e2cab)
![image-24](https://github.com/winstxnhdw/lc-hax/assets/158114082/b160766c-46d2-47ff-ac0e-263a63706fd7)

https://clips.twitch.tv/SullenSassyEmuTheTarFu-ZMVG1WymdLd6lWD5 just a lil example of what it can do.
Would this be possible to replicate outside of antikick?

**Comment by winstxnhdw** [comment]
It's not immediately obvious to me how this can be done.

**Reply by Totoqoe** [reply]
I can at least state the conditions that are needed for this to happen with antikick.
1. Antikick is enabled.
2. You must join through a lobby ID if you connect to a slot that used to be taken by another player.
3. If joining through a steam ID or through the rejoin, you must take a slot that no one else has used before (signified by the name "Player #x")
4. No one else should join after you, because when a new player joins, your character model loads in. However, players are allowed to leave as long as they don't rejoin.

**Reply by Totoqoe** [reply]
I hope I don't sound dumb but do ya think this has anything to do with `PlayerLoadedServerRpc` ?

**Reply by winstxnhdw** [reply]
Possibly.

**Reply by winstxnhdw** [reply]
Doesn't seem like it's `PlayerLoadedServerRpc`.
